app/2024/code-of-conduct/page.jsx

```diff
@@ -255,7 +255,7 @@ export default function CodeOfConduct() {
         </p>
         <ul className="list-disc list-inside md:list-inside">
           <li>Hassan Kibirige, Chairperson - <span className="font-bold">has2k1@gmail.com</span></li>
```

Comment:
given these are `email addresses` it would be nice to add them as links to open email boxes..

for example;

`<Link href="mailto:yourname@example.com ">yourname@example.com</Link>`

Reply:
Had thought of that at first but didn't think it was ideal that's why I just placed them that way. But that's alright

Reply:
should I also keep them bold now that we're using the "mailto" link

Reply:
sure yes please, given we want them to stand out on the page so that they arent missed by someone looking for them,

the font style can as well be applied to the link

Reply:
also given its a link a classname of `hover:underline` would be great to the same link

Reply:
![Screenshot from 2024-05-23 18-58-54](https://github.com/PyconUg/ug.pycon.org/assets/57962908/a0491364-47fc-460b-bb0b-0655174d9e92)
anyother suggestions you'd want to make??

Reply:
none on my end...

have you any?